src/Controllers/SubmitControllerCreate.php

```diff
@@ -88,4 +117,47 @@ function ($value)
 
 		return true;
 	}
+
+	/**
+	 * Check the CMS Version
+	 *
+	 * @return  false on failiure else the CMS Version
+	 *
+	 * @since   1.0
+	 */
+	private function checkCMSVersion($data)
+	{
+		$dotchecks = explode('.' $data);
+
+		// We ever use 2 dots and 3 parts in our CMS Version
+		if (count($dotchecks) != 3)
+		{
+			return false;
+		}
+
+		// The pulugin is installed since 3.5.0 other CMS Versions can't have this plugin installed
+		if (version_compare($data, '3.5.0', '<'))
```

Comment:
Yes i have updated that as well here: https://github.com/joomla-extensions/jstats-server/commit/8d56b80244e97243c1a6edd450704cdef84d0529 Thanks.